cachito/workers/tasks/rubygems.py

```diff
@@ -0,0 +1,16 @@
+from cachito.workers import nexus
+from cachito.workers.pkg_managers.rubygems import (
+    get_rubygems_hosted_repo_name,
+    get_rubygems_nexus_username,
+)
+from cachito.workers.tasks.celery import app
+
+
+@app.task
+def cleanup_rubygems_request(request_id):
```

Comment:
One more detail: functions from `tasks` modules must be imported in `tasks.__init__` https://github.com/containerbuildsystem/cachito/blob/a44fdb2246fcec26c1b7200706fab00a86d354a5/cachito/workers/tasks/__init__.py#L1-L7

and for the wildcard import to work, this module needs to define `__all__`